pysndfx/dsp.py

```diff
@@ -278,11 +332,15 @@ def vol(self, gain, type="amplitude", limiter_gain=None):
         if type in ["amplitude", "power", "dB"]:
             self.command.append(type)
         else:
-            raise InvalidEffectParameter("Type has to be dB, amplitude or power")
+            raise ValueError("Type has to be dB, amplitude or power")
```

Comment:
Why did you change from InvalidEffectParameter to ValueError?

Reply:
Well, after observing a bit how other librairies would report errors, I noticed that they almost *never* add their own type of errors, they just use the regular errors from standard python. I feel that custom errors are more used "inside" librairies to handle some problems, but the users themselves only get regular errors.

Reply:
Agree! 👍